src/SPC/command/BuildPHPCommand.php

```diff
@@ -319,6 +319,13 @@ private function parseRules(array $shared_extensions = []): int
             $rule |= BUILD_TARGET_EMBED;
             f_putenv('SPC_CMD_VAR_PHP_EMBED_TYPE=' . ($embed === 'static' ? 'static' : 'shared'));
         }
+        if ($this->getOption('build-frankenphp')) {
+            $rule |= BUILD_TARGET_FRANKENPHP;
+            if (!$this->getOption('enable-zts')) {
+                logger()->warning('FrankenPHP requires ZTS to work with multiple threads, the --enable-zts option will be enabled automatically!');
+                $this->input->setOption('enable-zts', true);
```

Comment:
I actually more prefer throwing wrong usage exception at `checkBeforeBuildPHP`. It's REAL wrong usage.